java/code/src/com/suse/manager/webui/utils/gson/BootstrapHostsJson.java

```diff
@@ -57,8 +61,30 @@ public BootstrapHostsJson(String hostIn, Integer portIn, String userIn,
         port = String.valueOf(portIn);
         user = userIn;
         password = passwordIn;
-        activationKeys = StringUtils.isEmpty(activationKey) ?
-                new ArrayList<>() : Arrays.asList(activationKey);
+        activationKeys = StringUtils.isEmpty(activationKey) ? new ArrayList<>() : Arrays.asList(activationKey);
```

Comment:
Nitpick: instead of `new ArrayList<>()` one could use `Collections.emptyList()`.

Reply:
Good point. Changed.